src/haz3lweb/Grading.re

```diff
@@ -350,7 +354,11 @@ module ImplGradingReport = {
               @ Option.to_list(
                   report.test_results
                   |> Option.map(test_results =>
-                       TestView.test_bar(~inject, ~test_results, -1)
+                       TestView.test_bar(
+                         ~inject,
+                         ~test_results,
+                         5 + num_hidden_bugs,
+                       )
                      ),
                 ),
             ),
```

Comment:
Pass num_hidden_bugs through functions.

Reply:
instead of passing in num_hidden_bugs directly, let's use SchoolExercise.{idx_of_pos/pos_of_idx} and pass in the exercise, so we aren't hard coding an indexing scheme everywhere that might change in the future